docs/observability.md

```diff
@@ -14,44 +14,45 @@ The instrumentation is **experimental** - volume and semantics of the telemetry
 
 To enable the instrumentation:
 
-1. Set instrumentation feature-flag using one of the following options:
-
-   - set the `OPENAI_EXPERIMENTAL_ENABLE_OPEN_TELEMETRY` environment variable to `"true"`
-   - set the `OpenAI.Experimental.EnableOpenTelemetry` context switch to true in your application code when application
-     is starting and before initializing any OpenAI clients. For example:
-
-     ```csharp
-     AppContext.SetSwitch("OpenAI.Experimental.EnableOpenTelemetry", true);
-     ```
-
-2. Enable OpenAI telemetry:
+1. Enable experimental OpenAI telemetry:
```

Comment:
nit: Add some note about what is experimental means here. Its good to call out that the name of span/metric and each attributes are subject to change /similar.

Reply:
It's called out in the paragraph before this, I'll expand it a bit